env0/resource_project_test.go

```diff
@@ -84,3 +85,84 @@ func TestUnitProjectInvalidParams(t *testing.T) {
 
 	runUnitTest(t, testCase, func(mockFunc *client.MockApiClientInterface) {})
 }
+
+func TestUnitProjectResourceDestroyWithEnvironments(t *testing.T) {
+	resourceType := "env0_project"
+	resourceName := "test"
+	accessor := resourceAccessor(resourceType, resourceName)
+
+	project := client.Project{
+		Id:          "id0",
+		Name:        "name0",
+		Description: "description0",
+	}
+
+	environment := client.Environment{}
+
+	t.Run("Success With Force Destory", func(t *testing.T) {
+		testCase := resource.TestCase{
+			Steps: []resource.TestStep{
+				{
+					Config: resourceConfigCreate(resourceType, resourceName, map[string]interface{}{
+						"name":          project.Name,
+						"description":   project.Description,
+						"force_destroy": true,
+					}),
+					Check: resource.ComposeAggregateTestCheckFunc(
+						resource.TestCheckResourceAttr(accessor, "id", project.Id),
+						resource.TestCheckResourceAttr(accessor, "name", project.Name),
+						resource.TestCheckResourceAttr(accessor, "description", project.Description),
+						resource.TestCheckResourceAttr(accessor, "force_destroy", "true"),
+					),
+				},
+			},
+		}
+
+		runUnitTest(t, testCase, func(mock *client.MockApiClientInterface) {
+			mock.EXPECT().ProjectCreate(client.ProjectCreatePayload{
+				Name:        project.Name,
+				Description: project.Description,
+			}).Times(1).Return(project, nil)
+
+			gomock.InOrder(
```

Comment:
no point to use in order for 1 check

Reply:
Fixed.
https://github.com/env0/terraform-provider-env0/pull/263/commits/f850c770528f357af8ad4654d4dbf151970ccaa7